File: Data Structures & Algorithms/counting-bits/submission-0.java
class Solution {
    public int[] countBits(int n) {
        int[] ans=new int[n+1];
        for(int i=0;i<=n;i++){
            int currAns=0;
            for(int j=0;j<32;j++){
                if((1<<j & i)!=0)currAns++;
            }
            ans[i]=currAns;
        }
        return ans;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        conList = {}
        ans = 0
        for i in nums:
            conList[i] = i
        for i in nums:
            elm = i
            while (elm - 1) in conList:
                elm -= 1
                if conList[elm] != elm:
                    break;
            conList[i] = conList[elm]
            ans = max(ans, i - conList[i]+1)
        return ans
